snap/local/launchers/launcher.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+HOME="$(getent passwd "${USER}" | cut -d: -f6)"
```

Comment:
Just curious: Why do you need this?

Reply:
I see you've merged already but this is because, by default, within the snap's runtime `$HOME` is set to another folder. SAM needs access to the user's `~/.aws` (granted by the `personal-files` plug) so we need to set `$HOME` to the proper folder :)